examples/dynamoathenamigration/migration.go

```diff
@@ -410,6 +411,10 @@ func (t *task) fromS3ToChan(ctx context.Context, dataObj dataObjectInfo, eventsC
 			return false, trace.Wrap(err)
 		}
 
+		if ev.GetID() == "" {
+			ev.SetID(uuid.New().String())
```

Comment:
Can the event ingest appear to fail in such a way that we end up resubmitting the same event with a different UUID?

Reply:
I have extended validation in https://github.com/gravitational/teleport/pull/29193/commits/8ca8c403bf2853dabdb6716dbfe5ce6db06d533c so all errors that publisher right now checks, will be handled.
It can still theoretically happen that event will fail and we will end up resubmitting event with different UUID, however it's very unlikely:
- event without UUID is very rare
- when running migration on internal tenants, the only error that happen is validation error

Moreover ending up with duplicated event is not that tragic.